copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,857 @@
+{
```

Comment:
<p>Is there more context around what is in "arrays"? Is it a dataframe or just an array of values? What data values are we displaying here, as opposed to what's in arr_values[n+1]?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Good point. I've updated the description. </p>